source/features/view-raw-file.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const createRawUrl = (href: string) => href.replace(/blob\//, 'raw/');
+
+function init() {
+	const fileActionsGroups: HTMLElement[] = select.all(
+		'.file-header .file-actions > div.BtnGroup'
+	);
+
+	for (const group of fileActionsGroups) {
```

Comment:
You can inline the select in here, on one line. 

Instead of selecting the group, select the link you’ll need on the next line with `[href*='/blob/']` and then use `link.after()` to append our link